gto-support-db/src/main/java/org/ccci/gto/android/common/db/Expression.java

```diff
@@ -760,4 +800,89 @@ public Unary createFromParcel(@NonNull final Parcel in, @Nullable final ClassLoa
             }
         }
     }
+
+    public static class Aggregate extends Expression {
+        static final String COUNT = "COUNT";
+        static final String MAX = "MAX";
+        static final String MIN = "MIN";
+        static final String SUM = "SUM";
+
+        @NonNull
+        private final String mOp;
+        @NonNull
+        private final Expression mExpr;
```

Comment:
Hmm, I was thinking this `mExpr` was to be used for queries like

````
SELECT ...
FROM ...
GROUP BY ...
HAVING count(select ... from ... where ...) > 1;
````

If this is not going to be useful, I can also get rid of the methods taking in parameters.